### PR TITLE
Implement GET /locations/{locationId}/services

### PR DESCRIPTION
**Pull request by kalcicd**

### CODE REVIEW

File: src/db/awsEs/compound-request-dao.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+const getServicesByLocationId = async (queryParams) => {
+  const client = Client(clientOptions());
+  const locationRes = await client.search({
+    index: 'locations',
+    body: buildIdQueryBody(queryParams, 'locations'),
+  });
+
+  const { _source: rawLocation } = locationRes.hits.hits[0];
+  const serviceIds = [];
+  _.forEach(rawLocation.relationships.services.data, (value) => serviceIds.push(value.id));
```
Comment (Ruefa) [inline src/db/awsEs/compound-request-dao.js]: Looks like you are just trying to get a list of `id`. Using `_.map()` would be better https://lodash.com/docs/4.17.15#map

File: src/db/awsEs/compound-request-dao.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import { clientOptions } from 'db/awsEs/connection';
+
+const buildIdQueryBody = (queryParams, type) => {
+  const q = esb.boolQuery();
+  const id = (type === 'locations') ? queryParams.locationId : queryParams.serviceId;
+  q.must(esb.matchQuery('id', id));
+  return esb.requestBodySearch().query(q).toJSON();
+};
+
+const buildBulkIdQueryBody = (ids, type) => {
```
Comment (Ruefa) [inline src/db/awsEs/compound-request-dao.js]: JSDoc for these two functions

**Review by Ruefa — CHANGES_REQUESTED** [review]

**Review by Ruefa — APPROVED** [review]

**Review by Ruefa — CHANGES_REQUESTED** [review]
Looks like there are merge conflicts now

**Comment by kalcicd** [comment]
fixed the conflict and also caught a bug with the serializer

**Review by tsoliangwu0130 — APPROVED** [review]

**Pull request merged**